### PR TITLE
Add changes to src_sycl which reflect equivalent changes in CUDA src

### PR DESCRIPTION
**Pull request by joeatodd**
All the changes in this PR have effectively already been reviewed in [this PR](https://github.com/codeplaysoftware/cuda-to-sycl-nbody/pull/6), as they are the same as on the CUDA side, but I forgot to run `dpct` and stage these changes.

### CODE REVIEW

**Comment by joeatodd** [comment]
I have just added a single functional change to the original CUDA code which I had already committed on the SYCL side - switching `glm::exp` and `glm::pow` for `std::exp` and `std::pow`. DPC++ complained about integer power with the `glm::` versions.

File: src_sycl/renderer_gl.cpp [hunk 1]
```diff
@@ -297,4 +310,60 @@ void RendererGL::render(glm::mat4 proj_mat, glm::mat4 view_mat) {
...
+
+   std::vector<float> result(halfwidth);
+   std::generate(result.begin(), result.end(), sigma_fun);
+
+   // Normalize the Gaussian kernel
+   float halfnorm = std::accumulate(result.begin() + 1, result.end(), 0.0);
+   float norm = 2 * halfnorm + result[0];
+
+   std::transform(result.begin(), result.end(), result.begin(),
+                  [&norm](auto &val) { return val / norm; });
```
Comment (Naghasan) [inline src_sycl/renderer_gl.cpp]:
```suggestion
                  [norm](auto val) { return val / norm; });
```

not sure why using references here

**Review by Naghasan — APPROVED** [review]
LGTM, just a small nitpick

File: src_sycl/renderer_gl.cpp (hunk 1 above)
Comment (joeatodd) [reply src_sycl/renderer_gl.cpp]: good point thanks

**Pull request merged**